drivers/note/noteram_driver.c

```diff
@@ -419,6 +569,15 @@ void sched_note_add(FAR const void *note, size_t notelen)
   spin_lock_wo_note(&g_noteram_lock);
 #endif
 
+  if (!g_noteram_info.ni_enable)
```

Comment:
can we change to no overwrite semantics instead of oneshot? so we need have enable/disable status.